src/data_source/storage/sql.rs

```diff
@@ -3259,16 +3236,16 @@ pub mod testing {
         
             CREATE TABLE {name}
             (
-                pos LTREE NOT NULL, 
+                path integer[] NOT NULL, 
                 created BIGINT NOT NULL,
                 hash_id INT NOT NULL REFERENCES hash (id),
                 children INT[],
                 children_bitvec BIT(8),
                 index JSONB,
                 entry JSONB 
             );
-            ALTER TABLE {name} ADD CONSTRAINT {name}_pk PRIMARY KEY (pos, created);
-            CREATE INDEX {name}_path ON {name} USING GIST (pos);"
+            ALTER TABLE {name} ADD CONSTRAINT {name}_pk PRIMARY KEY (path, created);
+            CREATE INDEX {name}_created ON {name} (path);"
```

Comment:
This seems like a typo, we have `_created` in the name but `(path)` in the index. Also, do we still need this index if the query now uses the primary key?

Reply:
addressed in `8d2acd8e17fe00b04015b8355bed91517605e1ad`

Reply:
Yeah the query planner uses this index for some individual queries